RELEASE/scripts/sl_ascend.ash

```diff
@@ -7002,6 +7004,9 @@ boolean L11_unlockEd()
 	if(get_property("sl_tavern") != "finished")
 	{
 		print("Uh oh, didn\'t do the tavern and we are at the pyramid....", "red");
+		// Force the Tavern since we need to do it anyways, and it may get us some tangles.
+		forceTavern = true;
+		L3_tavern();
```

Comment:
Don't directly call `L3_tavern` from here, once you switch `forceTavern` in to a property it should just automatically happen if it can.